docs/quickstart.rst

```diff
@@ -123,6 +75,32 @@ Once you have a recent Python 3 version of ``pip``, you should be able to upgrad
     $ pip3 install --upgrade setuptools
 
 
+Dependency 4: cryptography and cryptoconditions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+BigchainDB(server and driver) also depends on `cryptography`_ and `cryptoconditions`_.
+
+* `cryptography`_ depends on `libssl`_, `libcrypto`_ which also depends on (`Python development library and header files`_).
+* `cryptoconditions`_ depends on `PyNaCl`_ (`Networking and Cryptography library`_) which depends on ``ffi.h``.
```

Comment:
The BigchainDB Python driver also depends on `cryptoconditions` which depends on.  `cryptography` and  `PyNaCl` ...